plugins/modules/terraform.py

```diff
@@ -564,7 +564,7 @@ def main() -> None:
             plan_file_to_apply = plan_file
             if not plan_file:
                 f, new_plan_file = tempfile.mkstemp(suffix=".tfplan")
-                module.add_cleanup_file(new_plan_file)
+                # module.add_cleanup_file(new_plan_file)
```

Comment:
The comment should be removed from this ?!

Reply:
yes, good catch!!! (leftover from debugging)